sr_data_visualization/uis/e_visualizer_test.ui

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we need this file?

Reply:
In fact there are several files ".ui". Please delete them and only leave the final one

Reply:
Removed all but the one that's being used